mapiproxy/services/plugins/dovecot/.gitignore

```diff
@@ -0,0 +1,22 @@
+.deps
+.libs
+AUTHORS
```

Comment:
This file are required for `./autogen.sh`:

    Makefile.am: error: required file './NEWS' not found
    Makefile.am: error: required file './README' not found
    Makefile.am: error: required file './AUTHORS' not found
    Makefile.am: error: required file './ChangeLog' not found
````